constants.js

```diff
@@ -252,4 +259,11 @@ const constants = {
     onlyOwnerAllowed: ['bucketDeletePolicy', 'bucketGetPolicy', 'bucketPutPolicy'],
 };
 
+// Some apis may need a custom body length limit :
+constants.apisLengthLimits = {
```

Comment:
should be set in `constants` definition directly?
(not sure it is worth setting variables for "oneMegaBytes" and "twoMegaBytes")